engine/runtime/src/test/java/org/enso/interpreter/test/MetaIsATest.java

```diff
@@ -0,0 +1,118 @@
+package org.enso.interpreter.test;
+
+import java.io.ByteArrayOutputStream;
+import java.net.URI;
+import java.nio.file.Paths;
+import java.util.Map;
+import org.enso.polyglot.RuntimeOptions;
+import org.graalvm.polyglot.Context;
+import org.graalvm.polyglot.Engine;
+import org.graalvm.polyglot.Language;
+import org.graalvm.polyglot.Source;
+import org.graalvm.polyglot.Value;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+import org.junit.Before;
+import org.junit.Test;
+
+public class MetaIsATest {
+  private Context ctx;
+  private Value isACheck;
+
+  @Before
+  public void prepareCtx() throws Exception {
+    Engine eng = Engine.newBuilder()
+      .allowExperimentalOptions(true)
+      .logHandler(new ByteArrayOutputStream())
+      .option(
+        RuntimeOptions.LANGUAGE_HOME_OVERRIDE,
+        Paths.get("../../distribution/component").toFile().getAbsolutePath()
+      ).build();
+    this.ctx = Context.newBuilder()
+      .engine(eng)
+      .allowIO(true)
+      .allowAllAccess(true)
+      .build();
+    final Map<String, Language> langs = ctx.getEngine().getLanguages();
+    assertNotNull("Enso found: " + langs, langs.get("enso"));
+
+    final URI facUri = new URI("memory://choose.enso");
+    final Source facSrc = Source.newBuilder("enso", """
```

Comment:
```suggestion
    final URI uri = new URI("memory://choose.enso");
    final Source src = Source.newBuilder("enso", """
```

Seems like the names are leftovers from some other file? Potentially confusing when we'll be reading this in the future. I'd rename to something more generic.